packages/devtools_extensions/example/app_that_uses_foo/test/dart_test/example_dart_test.dart

```diff
@@ -0,0 +1,26 @@
+// Copyright 2024 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:dart_foo/dart_foo.dart';
+import 'package:test/test.dart';
+
+// This test can be run to verify that the `package:foo` DevTools extension
+// loads properly when debugging a Dart test target with DevTools.
+//
+// This doubles as a test to make sure the DevTools extension loads properly
+// when the test target is in a subdirectory of 'test/'.
+//
+// To test this, run the following command and copy the VM service URI to
+// connect to DevTools:
+//
+// dart run test/dart_test/example_dart_test.dart --start-paused
```

Comment:
What's using this `--start-paused` flag?

Reply:
so we can open DevTools connected to the test and see the extension